src/libinjector/linux/methods/linux_read_file.c

```diff
@@ -118,7 +119,8 @@ bool init_read_file_method(injector_t injector, const char* file)
     FILE* fp = fopen(file, "wb");
     if (!fp)
     {
-        fprintf(stderr, "Could not open (%s) for writing\n", file);
+        fprintf(stderr, "Could not open (%s) for writing: errno=%d\n", file, errno);
```

Comment:
how about using perror?

Reply:
`stderror` is being used in the `PRINT_DEBUG` statement ( the next line ), should I remove that line and print the same info using `perror` in this line?

Reply:
Oh, I did not see that;) I think one line is enough, so you can delete PRINT_DEBUG.

Reply:
done ( used `strerror` with `fprintf` )